plugins/module_utils/ndb/time_machines.py

```diff
@@ -20,6 +20,9 @@ def __init__(self, module):
             "desc": self._build_spec_desc,
             "schedule": self._build_spec_schedule,
             "auto_tune_log_drive": self._build_spec_auto_tune_log_drive,
+            "cluster": self._build_spec_cluster,
```

Comment:
please create seperate builder called get_spec_for_data_access_management() and update _build_spec_methods in that and call get_spec(). As TimeMachine is creating spec for different purposes in different modules like databases

Reply:
check builders (get_spec* methods) of  DbServerVM class for way to seperate builders as per the purpose